src/components/TemplatesPage/NavItem/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const getNavItemClasses = (
```

Comment:
Rename the file to utils.ts, since this is a helper function, not a constant value.